drivers/include/at86rf2xx.h

```diff
@@ -32,6 +32,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+#include "at86rf2xx_registers.h"
```

Comment:
I don't like this, it breaks the separation of the public API and the internal register addresses.